modules/terraform/output.go

```diff
@@ -100,11 +100,24 @@ func parseMap(m map[string]interface{}) (map[string]interface{}, error) {
 			}
 			result[k] = nestedMap
 		case []interface{}:
-			nestedList, err := parseListOfMaps(vt)
-			if err != nil {
-				return nil, err
+			if len(vt) == 0 {
+				result[k] = []interface{}{}
+			} else {
+				switch vt[0].(type) {
+				case map[string]interface{}:
+					nestedList, err := parseListOfMaps(vt)
+					if err != nil {
+						return nil, err
+					}
+					result[k] = nestedList
+				default:
+					nestedList, err := parseListOutputTerraform(vt, k)
```

Comment:
Is `parseListOutputTerraform` the correction function to be using here? Should there perhaps be a `parseList` instead?